src/tests/ftest/util/agent_utils.py

```diff
@@ -65,8 +161,27 @@ def run_agent(basepath, server_list, client_list=None):
     sessions = {}
     user = getpass.getuser()
 
-    # if empty client list, 'self' is effectively client
-    client_list = include_local_host(client_list)
+    # if empty client list, 'test' is effectively client
+    if client_list == None:
```

Comment:
(pylint-singleton-comparison) Comparison to None should be 'expr is None'

Reply:
Fixed.